src/stories/NewButton.stories.tsx

```diff
@@ -145,9 +145,14 @@ export const caretButton = ({...args}: ButtonProps) => {
 
 export const blockButton = ({...args}: ButtonProps) => {
   return (
-    <Button {...args} sx={{width: '100%'}}>
-      Block
-    </Button>
+    <>
+      <Button {...args} sx={{width: '100%', mb: 2}}>
+        Block
+      </Button>
+      <Button leadingIcon={EyeIcon} trailingIcon={TriangleDownIcon} {...args} sx={{width: '100%'}}>
+        Watch
+      </Button>
+    </>
```

Comment:
Did you combine changes somehow?

Reply:
haha yeah, I merged that branch here to see what it might look like 😇  (don't intent to ship them together though!)